src/components/search/tests/SearchProgramCard.test.jsx

```diff
@@ -81,7 +77,7 @@ describe('<SearchProgramCard />', () => {
     const { container } = renderWithRouter(<SearchProgramCardWithAppContext {...defaultProps} />);
 
     expect(screen.getByText(PROGRAM_TITLE)).toBeInTheDocument();
-    expect(screen.getByText(PROGRAM_AUTHOR_ORG.key)).toBeInTheDocument();
+    expect(screen.getAllByText(PROGRAM_AUTHOR_ORG.key).length).toBeGreaterThan(0);
```

Comment:
[curious] Can you explain the rationale for this change?

Reply:
The test was failing after switch to `react-lines-ellipsis`, as the text within the "LinesEllipsis" component is being broken into multiple parts and being rendered multiple times in the view. **This is no longer an issue after switch to Paragon `Truncate` & I have reverted the change.**

**Explanation**: `react-lines-ellipsis` employs multiple hidden nodes to calculate which parts of the text to show or hide. It breaks down the text into smaller units uses this to determine the ellipsis effect. This was resulting in multiple parts of the DOM containing the same text string, which is why getByText was throwing an error as it was finding multiple instances of the expected text.